circle.yml

```diff
@@ -7,6 +7,15 @@ test:
     - gulp -v
   override:
     - npm test # Run the package and docs test suite
-    - npm run build:package # Run the release process
+    - npm run build
   post:
-    - ls -agolf dist/ # Ensure that build:package worked
\ No newline at end of file
+    - ls -agolf dist/ # Ensure that build:package worked
+
+deployment:
+  npm:
+    branch: master
```

Comment:
We do use git tags for each of our releases. At the moment that only happens when we publish our release notes, hence pointing the at `master`. I'd like to change this up in the future where we remove the `release` branch and target releases with tags instead. 

Hope that makes sense!